### PR TITLE
Fix highlighter

### LINKED ISSUE #207: highlighter issues
highlighter is having some bugs around bit-dev:

- some components don't get highlighted (forwardRef, mdx, etc)
- some components hide the highlighter

for example
image

### PR DESCRIPTION
**Pull request by KutnerUri**
closed #207

- fix highlighter missing DocPage
- quickfix z-index
- bump highlighter with support for forwardRefs

### CODE REVIEW

File: docs/ui/docs/docs.module.scss [hunk 1]
```diff
@@ -17,41 +17,3 @@
...
-      width: 0;
-    }
-  }
-}
-
-.primaryLinks {
-  border-bottom: 1px solid var(--bit-border-color-lightest, #ededed);
-  font-size: var(--bit-p-xs, 14px);
-  padding: 16px 0;
-}
```
Comment (KutnerUri) [inline docs/ui/docs/docs.module.scss:57]: unused

File: docs/ui/pages/doc-page/doc-page.tsx [hunk 2]
```diff
@@ -53,9 +53,9 @@ export function DocPage({ route, index, children, plugins = [] }: DocPageProps)
   }, [window?.location.hash]);
 
   return (
-    <Suspense fallback={<div />}>
-      <DocPageContext.Provider value={{ index, route }}>
-        <Page title={`${route.title} | Bit`} description={pageDescription} className={styles.docsPage}>
+    <Page title={`${route.title} | Bit`} description={pageDescription} className={styles.docsPage}>
```
Comment (KutnerUri) [inline docs/ui/pages/doc-page/doc-page.tsx:56]: reorganize - move `<Page>` up

File: workspace.jsonc [hunk 3]
```diff
@@ -109,7 +109,7 @@
         "@teambit/mdx.ui.mdx-layout": "0.0.479",
         "@teambit/react.eslint-config-bit-react": "0.0.451",
         "@teambit/react.templates.themes.theme": "0.0.3",
-        "@teambit/react.ui.component-highlighter": "0.0.496",
+        "@teambit/react.ui.component-highlighter": "0.0.506",
```
Comment (KutnerUri) [inline workspace.jsonc:112]: fix - forward ref

File: workspace.jsonc [hunk 4]
```diff
@@ -90,7 +90,7 @@
         "@teambit/design.ui.input.toggle": "1.0.7",
         "@teambit/design.ui.message-card": "1.0.6",
         "@teambit/design.ui.panels.tabs": "1.0.6",
-        "@teambit/design.ui.sidebar.sidebar": "1.9.57",
+        "@teambit/design.ui.sidebar.sidebar": "1.10.0",
```
Comment (KutnerUri) [inline workspace.jsonc:93]: fix - z-index

**Review by odedre — APPROVED** [review]

**Pull request merged**